CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -25,6 +25,7 @@
 using Microsoft.Xna.Framework.Content;
 using Microsoft.Xna.Framework.Graphics;
 using Microsoft.Xna.Framework.Input;
+using static CardsCashCasino.Manager.PlayerManager;
```

Comment:
manager classes should never be static. The C# convention says: managers are classes, utils are static objects. Just make an instance of PlayerManager in the class.

Conversely, if you want to statically reference something NEVER do this. Always call PlayerManager.XXXX.

If this can be static and does not need to be a class, then make it a Util class.